cdn/dev/js/kmlive.js

```diff
@@ -135,7 +135,7 @@ function loaded(){
       } else {
         window.location.href = '/mac/download.php';
       }
-    }else if (platform == 'Windows') {
+    }else if (platform == 'Windows' || platform == 'Linux') {
       if (language && keyboard) {
```

Comment:
Maybe Linux should get its own block like mac ?
```
} else {
  link = '/linux/download.php';
}
```

Reply:
done